README.md

```diff
@@ -1,14 +1,63 @@
-# Java Client Library for Ready! API TestServer
+# ReadyApi4J - a Java library for API tests
 
-This repository contains source file for the Java Client library for 
+The ReadyApi4J library lets Java developers use the API testing functionality in SoapUI from Java code. In spite of its somewhat unfortunate name, which is there for historical reasons, SoapUI supports many protocols in addition to SOAP, notably REST, JDBC and JMS.
+
+ReadyApi4J builds *test recipes*, which describe API tests to be executed.
+
+Under the hood ReadyApi4J uses a JSON format, but there's no need to learn it, because you typically create and run test recipes using a fluent Java API. Nor do you need to install SoapUI or any other software to be able to execute recipes.
+
+## Running local SoapUI tests on your machine
+
+The support for local SoapUI tests is ready to use but is not released yet. Therefore, until the upcoming 2.0.0 release, there are no public artefacts uploaded to the Maven repos. However if you build ReadyApi4J locally, with Maven, you can start using it in the following way.

```

Comment:
This part confuses me a bit... We have no support for local tests yet so if you want to use it locally, do this....